### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by dcolleoni**
Describe Parameter Reports
Add links to the Practical use cases in Template readme

### CODE REVIEW

**Comment by nnikolopoulos** [comment]
I suppose that also the "v1.0.0" EnvironmenrVariable should be changed to v1.1.0 in the CalculationEngine notebook:

image

File: ifrs17-template/Import/CloseImportTemplate.ipynb [hunk 1]
```diff
@@ -38,7 +38,7 @@
             "source": [
                 "All imports are triggered via the <code>Import</code> command. Several options can be fluently appended to customize the import action:",
                 "\n- <code>FromFile()</code> : accepts a string as input (eg. \"../Files/TransactionalData/Openings_CH_2020_12.csv\"). It corresponds the file to be imported complete with the path",
-                "\n- <code>WithFormat()</code> : accepts a string as input (eg. \"Cashflow\" or \"Actual\"). It is used to trigger the desired import logic for the current file. Full list of avalaible ImportFormat can be found [here](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/Constants/Consts#import-formats)",
+                "\n- <code>WithFormat()</code> : accepts a string as input (eg. \"Cashflow\" or \"Actual\"). It is used to trigger the desired import logic for the current file. Full list of avalaible ImportFormat can be found [here](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/Constants/Consts#import-formats)",
```
Comment (nnikolopoulos) [inline ifrs17-template/Import/CloseImportTemplate.ipynb:41]: Typo? ImportFormat --> ImportFormats

File: ifrs17-template/InputFormatDescription.ipynb [hunk 2]
```diff
@@ -39,17 +39,17 @@
...
+                "\n- [AocType](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/DataModel/DataStructure#aoc-type) : entered with its SystemName,",
+                "\n- [Novelty](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/DataModel/DataStructure#novelty) : entered with its SystemName,",
                 "\n- AccidentYear : relevant for Data Node with Liability Type: Liability for Incurred Claims.",
                 "\n",
-                "\nAll valid combinations of AocType and Novelty can be found in [AocConfiguration](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/DataModel/DataStructure#aoc-configuration) with DataType Optional or Mandatory.",
+                "\nAll valid combinations of AocType and Novelty can be found in [AocConfiguration](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/DataModel/DataStructure#aoc-configuration) with DataType Optional or Mandatory.",
                 "\n<br>The cash flow is entered in the columns named <code>ValuesN</code> where N goes from 0 to the lenght of cash flow. The Value0 corresponds to January of the Year entered in the **Main** section for every combination of AocType and Novelty. "
```
Comment (nnikolopoulos) [inline ifrs17-template/InputFormatDescription.ipynb:53]: Typo: lenght --> length

File: ifrs17-template/InputFormatDescription.ipynb [hunk 3]
```diff
@@ -74,13 +74,13 @@
             "source": [
                 "An Actuals file is named **Actuals** *\\_<reportingNode\\>* *\\_<year\\>* *\\_<month\\>* and is composed of two sections: **Main** and **Actual**.",
                 "\n",
-                "\nThe **Main** section contains information such as: Reporting Node, Year, Month for which the data is being imported. This information is used to define the [partition](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/DataModel/DataStructure#partition) to which the data belong.",
+                "\nThe **Main** section contains information such as: Reporting Node, Year, Month for which the data is being imported. This information is used to define the [partition](https://portal.systemorph.cloud/project/ifrs17/env/v1.1.0/DataModel/DataStructure#partition) to which the data belong.",
```
Comment (nnikolopoulos) [inline ifrs17-template/InputFormatDescription.ipynb:77]: Typo? data is being imported --> data are being imported

**Review by nnikolopoulos — APPROVED** [review]

File: ifrs17-template/InputFormatDescription.ipynb (hunk 3 above)
Comment (dcolleoni) [reply ifrs17-template/InputFormatDescription.ipynb:77]: Thanks, both constructions are standard: https://www.merriam-webster.com/dictionary/data. Given we mostly use it as singular I also leave this as is.

**Comment by dcolleoni** [comment]
> I suppose that also the "v1.0.0" environmentName should be changed to v1.1.0 in the CalculationEngine notebook:
> 
> image

done

**Pull request merged**